src/components/modal.md

```diff
@@ -0,0 +1,57 @@
+Html Snippet: Markup for Dialog Box
+
+```html
+<div class="dg_modal dialog" id="my-accessible-dialog">
+
+  <!--
```

Comment:
please remove any comments in the example, web services won't care about them.

Reply:
done